README.md

```diff
@@ -977,6 +1003,7 @@ end
 - [@ezekg](https://github.com/ezekg)
 - [@0x0dea](https://github.com/0x0dea)
 
+
```

Comment:
did you use the rake task to generate the readme? cause I think it wouldn't add these empty lines, does it?